test/env/deploy/cluster.yaml

```diff
@@ -264,3 +264,10 @@ spec:
                 image: registry:5000/common-docker-third-party/contrail/centos-binary-swift-account:train-2005
               - name: swiftAccountServer
                 image: registry:5000/common-docker-third-party/contrail/centos-binary-swift-account:train-2005
+        commonConfiguration:
```

Comment:
why hostNetwork is needed? Move `commonConfiguration` before `serviceConfiguration` like in other components and remove hostNetwork from here it should be set at Manager level